actioncable.go

```diff
@@ -362,6 +408,7 @@ type Command struct {
 type CommandIdentifier struct {
 	// Channel is the name of the channel.
 	Channel string
+	RawData json.RawMessage
```

Comment:
does this actually need to be added to the public API too? Since I haven't used ActionCable this way I figure I'm probably just not understanding the use case, can you explain it for me?